README.md

```diff
@@ -203,14 +204,46 @@ Space      | Skip cutscenes, interact
 Ctrl + c   | Force Quit (from command-line)
 Ctrl + q   | Quit (ingame)
 
+## 5. Running The Longest Journey
+
+### 5.1. Required files ###
+
+You will need these elements from your game CDs or DVD :
+  * *TODO*
+  
+For digital distributions (Steam, GOG, demo from Steam or other sources),
+you can just add the game folder to ResidualVM without any changes.
+  
+### 5.2. Running the game ###
+
+1. Copy the necessary files to a folder on your Hard Drive
+2. Open ResidualVM
+3. Choose "Add Game"
+4. Select the folder you created in step 1
+5. Click Start
+
+### 5.3. Input controls ###
+
+The mouse is used to interact with objects and menu elements.
+
+Available keyboard shortcuts:
+
+Key        | Binding
+---------- | ------------------------------
+Escape     | Skip video sequence or current line of dialogue, skip time if *Time Skip* option is enabled
+Alt + x    | Quit
+Alt + q    | Quit
```

Comment:
Alt + x and Alt + q are platform specific. I'm not sure it's a good idea to document them here.

Reply:
Sorry, my bad – these are not platform specific shortcuts, but from the manual, just forgot to remove `Alt + x` and `Alt + q` after I realized they are not implemented in ResidualVM.

*However*, ESC is working quite fine, and with the settings now merged it properly doesn't work for time skipping (but still skips movies and dialogues) when the time skip isn't on, so I guess it's worth mentioning.

For reference, my original PR included (based on the manual):
```
Key        | Binding
---------- | ------------------------------
Escape     | Skip video sequence or current line of dialogue, skip time if *Time Skip* option is enabled
F1         | Diary Menu
F2         | Save game
F3         | Load game
F4         | Conversation Log
F5         | April's Diary (initially disabled)
F6         | Video replay
F7         | Game settings
F8         | Save a screenshot to your local game directory
F9         | Toggle subtitles on and off
F10        | Quit game and return to to main menu
A          | Cycle back through inventory cursor items
S          | Cycle forward through inventory cursor items
I          | Inventory
P          | Pause the game
X          | Display all exits on current location
Page Up    | Scroll up in dialogues and in your inventory
Up arrow   | Scroll up in dialogues and in your inventory
Page Down  | Scroll down in dialogues and in your inventory
Down arrow | Scroll down in dialogues and in your inventory
Enter      | Select currently highlighted dialogue choice
1 – 9      | Select a dialogue choice
Alt + x    | Quit
Alt + q    | Quit
```